charts/newrelic-infrastructure/values.yaml

```diff
@@ -113,13 +113,13 @@ kubelet:
     requests:
       cpu: 100m
       memory: 150M
+  # -- Max number of scraper rerun when scraper runtime error happens
+  scraperMaxReruns: 4
```

Comment:
this still needs to be nested in the config right?

Reply:
reverted the change